tests/shopify_python/test_git_utils.py

```diff
@@ -211,6 +211,15 @@ def test_dont_include_uncommited_or_untracked_files(main_repo, python_file):
     assert git_utils.changed_python_files_in_tree(main_repo.working_dir) == [os.path.basename(python_file)]
 
 
+def test_get_uncommitted_python_files(main_repo, python_file):
```

Comment:
Here you're only testing with staging a file, we should also test unstaged files.